### PR TITLE
Remove <Overflow>, <OverflowLink>, <OverflowYFade> and replace instances with .co-break-*

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes https://github.com/openshift/console/pull/1031
After:
screen shot 2019-01-08 at 3 17 40 pm

### CODE REVIEW

**Comment by rhamilto** [comment]
@spadgett, please review

File: frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx [hunk 1]
```diff
@@ -167,15 +166,15 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
               <dt>Links</dt>
               { spec.links && spec.links.length > 0
                 ? spec.links.map((link, i) => <dd key={i} style={{display: 'flex', flexDirection: 'column'}}>
-                  {link.name} <OverflowLink value={link.url} href={link.url} />
+                  {link.name} <a className="co-break-word" href={link.url}>{link.url || '-'}</a>
```
Comment (spadgett) [inline frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx]: If we make this change, do we we want `co-break-all`?

Not new, but most other external links open in a new window. I'd think we'd want that here, too.

Comment (rhamilto) [reply frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx]: > If we make this change, do we we want co-break-all?

Lemme investigate.  It wasn't needed here, but was in #1032.

> Not new, but most other external links open in a new window. I'd think we'd want that here, too.

Good catch.  Will update.

Comment (rhamilto) [reply frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx]: Updated to decorate links as external and open them in a new window.  `.co-break-word` is correct here.  PTAnotherLook, @spadgett.

**Comment by rhamilto** [comment]
@spadgett, PTAnotherLook

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
```
ERROR in /go/src/github.com/openshift/console/frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx
(171,84): Property 'additionalClassName' does not exist on type 'IntrinsicAttributes & { href: any; text: any; }'.
```

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
Updated to remove all Overflow components.

**Comment by rhamilto** [comment]
@spadgett PTAL.

**Comment by rhamilto** [comment]
jenkins rebuild

**Pull request merged**